datahub/company/management/commands/sync_ch.py

```
@@ -112,20 +127,22 @@ def truncate_ch_companies_table():
     """
     cursor = connection.cursor()
     table_name = CompaniesHouseCompany._meta.db_table
+    logger.info('Truncating the %s table', table_name)
     query = f'truncate {table_name};'
     cursor.execute(query)
 
 
+def _create_ch_company(row_dict):
+    return CompaniesHouseCompany(**row_dict)
+
+
 class Command(BaseCommand):
     """Companies House sync command."""
 
     def handle(self, *args, **options):
         """Handle."""
         try:
             sync_ch(tmp_file_creator=tempfile.TemporaryFile, truncate_first=True)
-        except Exception as e:
-            with log_and_ignore_exceptions():
```

So what? It's already blown-up by this point and the exception chain will be there.

The old code was broken because `self.stderr.write(e)` takes a string and not an exception. Even if you fixed it, it would mess up the exit code because it was swallowing up any errors.